docker-compose.yml

```diff
@@ -28,6 +32,7 @@ services:
       - wait-for
       - db:5432
       - migrate
+      - celery-queues
```

Comment:
Hi @jbagot !!

I didn't know about pyinvoke, it's cool!

But I don't know how run `celer-queues` and `runserver` in parallel.

Could you help me, please?

Reply:
Ok! I got it. I'm gonna implement it.

Reply:
I've created a new celery service